app/uk/gov/hmrc/serviceconfigs/service/AlertConfigService.scala

```diff
@@ -110,21 +110,19 @@ object AlertConfigService {
         }
     }
 
-  def toRepos(sensuConfig: SensuConfig): Seq[Repo] =
+  def toRepoNames(sensuConfig: SensuConfig): Seq[String] =
```

Comment:
Should the return type now be `Seq[RepoName]`?
Same for `toAlertEnvironmentHandler`